Dockerfile

```diff
@@ -29,14 +29,15 @@ COPY ${REMOTE_SOURCE}/apis ./apis
 COPY ${REMOTE_SOURCE}/controllers ./controllers
 COPY ${REMOTE_SOURCE}/internal ./internal
 
+USER 0
 RUN make build
 
 
 #@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.8)
-FROM registry.ci.openshift.org/ocp/4.8:cli AS origincli
+FROM quay.io/openshift/origin-cli:4.8 AS origincli
 
 #@follow_tag(registry.redhat.io/ubi8:latest)
-FROM registry.ci.openshift.org/ocp/4.8:base
+FROM registry.access.redhat.com/ubi8
```

Comment:
suggest to use
```
registry.redhat.io/ubi8:8.5 
```

Reply:
this one is fine `Terms & conditions
The Red Hat Universal Base Image is free to deploy on Red Hat or non-Red Hat platforms and freely redistributable. S`